test/articles/flages.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable linebreak-style */
+require('dotenv').config();
+const chai = require('chai');
+const { describe, it } = require('mocha');
+const http = require('chai-http');
+const jwt = require('jsonwebtoken');
+const app = require('../../server/index');
+
+chai.use(http);
+chai.should();
+
+describe('When a user wants to flag an article ', () => {
+  it('should not be able to flag an article as inappropriate when no token', (done) => {
+    chai.request(app)
+      .post('/api/v1/articles/1')
+      .send({
+        flagId: 1,
+      })
+      .end((err, res) => {
+        res.should.have.status(401);
+        done();
+      });
+  });
+
+  const payload = {
+    id: 2,
+    first_name: 'Patrick',
+    last_name: 'Tunezerwane',
+    email: 'tp3@gmail.com',
+    password: '$2b$10$Atnw/KEDHvmcSdNTRWfMfOZIOOQFOIynwjiYqGGZx3xtemaF6NGe6',
+    gender: 'Male',
+    jobRole: 'Manager',
+    department: 'ICT',
+    address: 'Kigali',
+    is_admin: false,
+  };
```

Comment:
Move this to mockData file to keep the tests DRY!

Reply:
oh! thanks tests should be DRY for sure. Thank you